.github/actions/build/action.yml

```
@@ -0,0 +1,9 @@
+name: Build
+description: Run Build
+runs:
+  using: "composite"
+  steps:
+    - run: npm install --silent --unsafe-perm
```

Why did you use --unsafe-perm? It seems to me that runners are not containers
```suggestion
    - run: npm install --silent
```